src/applications/financial-status-report/actions/geographicMeansThreshold.js

```diff
@@ -4,31 +4,28 @@ import environment from 'platform/utilities/environment';
 
 export const getGMT = (dependents, year, zipCode) => {
   const CONTEXT_ROOT = '/income_limits/v1/limitsByZipCode';
-  const REMOTE_REQUEST_URL = `${
+  const REQUEST_URL = `${
     environment.API_URL
   }${CONTEXT_ROOT}/${zipCode}/${year}/${dependents}`;
-  // For testing locally -- borrowed from income_limits app
-  const LOCAL_REQUEST_URL = `https://api.va.gov/income_limits/v1/limitsByZipCode/${zipCode}/${year}/${dependents}`;
 
-  const REQUEST_URL = environment.isLocalhost()
-    ? LOCAL_REQUEST_URL
-    : REMOTE_REQUEST_URL;
+  // For testing locally if api db isn't populated
+  // const REQUEST_URL = `https://staging-api.va.gov/income_limits/v1/limitsByZipCode/${zipCode}/${year}/${dependents}`;
 
   return apiRequest(REQUEST_URL)
     .then(({ data }) => {
-      // incomeStatus is 150% of the GMT
-      const incomeStatus = data.gmtThreshold * 1.5;
-      // assetStatus is 6.5% of the GMT
-      const assetStatus = data.gmtThreshold * 0.065;
+      // incomeUpperThreshold is 150% of the GMT
+      const incomeUpperThreshold = data.gmtThreshold * 1.5;
+      // assetThreshold is 6.5% of the GMT
+      const assetThreshold = data.gmtThreshold * 0.065;
       // discressionaryStatus is 1.25% of the GMT
-      const discressionaryStatus = data.gmtThreshold * 0.0125;
+      const discressionaryIncomeThreshold = data.gmtThreshold * 0.0125;
```

Comment:
Discressionary should be Discretionary 🙊 discretionaryIncomeThreshold

Reply:
Since the thresholds are probably going to remain consistant what are your thoughts on converting them to constants?

```
const INCOME_UPPER_THRESHOLD = 1.5;
const ASSET_THRESHOLD = 0.065;
const DISCRETIONARY_INCOME_THRESHOLD = 0.0125;

const incomeUpperThreshold = data.gmtThreshold * INCOME_UPPER_THRESHOLD;
const assetThreshold = data.gmtThreshold * ASSET_THRESHOLD;
const discressionaryIncomeThreshold = data.gmtThreshold * DISCRETIONARY_INCOME_THRESHOLD;
```

Reply:
Spelling is hard 😅 commenting as I address these locally, and will resolve convo after next commit.